ingest-app/src/cmr/ingest/api/subscriptions.clj

```diff
@@ -60,6 +62,28 @@
            subscriber-id
            concept-id)))))
 
+(defn- search-gran-refs-with-sub-params
+  "Performs a granule search using the provided query parameters. If the query is no good,
+   we throw a service error."
+  [context params]
+  (try
+    (search/find-granule-references context params)
+    (catch Exception e
+      (errors/throw-service-error :bad-request
```

Comment:
The error should contain the specific error message returned from the search-app.